BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -67,6 +70,36 @@ private struct SearchableVaultListView: View {
 
     // MARK: Private Properties
 
+    /// A floating action button positioned at the bottom-right corner of the screen.
+    @ViewBuilder private var newLoginFloatingActionButton: some View {
+        VStack {
+            Spacer()
+            HStack {
+                Spacer()
+                Button {
+                    store.send(.addItemPressed)
+                } label: {
+                    Circle()
+                        .fill(Asset.Colors.buttonFilledBackground.swiftUIColor)
+                        .frame(width: 50, height: 50)
+                        .overlay(
+                            Asset.Images.plus.swiftUIImage
+                                .imageStyle(
+                                    .init(
+                                        color: Asset.Colors.buttonFilledForeground.swiftUIColor,
+                                        scaleWithFont: false,
+                                        width: 24,
+                                        height: 24
+                                    )
+                                )
+                        )
+                }
+                .accessibilityIdentifier("new-login-floating-action-button")
```

Comment:
❓ Curious, I didn't see this in the ticket, where did it come from? Our other accessibility identifiers are all pascal case, so just wanted to confirm.

Reply:
oh i was using this to get the button for testing... is that bad practice?

Reply:
Ah, the QA team also uses these for automated testing, but usually they specify the ID so the buttons match between platforms. This should probably match the add button's ID in the toolbar (`AddItemButton`) if I had to guess. 

Usually we use the accessibility label to get the button for testing, but the identifier works as well.

Reply:
It might be worth running this by QA to see what they prefer for the ID here

Reply:
i removed the id and added a label, i was using it incorrectly and didn't realize QA was using them.  sorry about the confusion here!